modules/local/markdups/meta.yml

```diff
@@ -0,0 +1,62 @@
+name: markdups
+description: Identify and mark duplicate reads ifrom alignment data
```

Comment:
```suggestion
description: Identify and mark duplicate reads from alignment data
```

Reply:
- Change made in https://github.com/nf-core/oncoanalyser/pull/67